pkg/internal/netolly/flow/deduper.go

```diff
@@ -91,9 +91,8 @@ func DeduperProvider(dd *Deduper) (pipe.MiddleFunc[[]*ebpf.Record, []*ebpf.Recor
 				// Before forwarding, unset the non-common fields of deduplicate flows.
 				// These values are not relevant after deduplication and keeping them
 				// would unnecessarily increase cardinality, as they could chaotically
-				// contain the different directions and interfaces.
+				// contain the different interfaces.
 				record.Id.IfIndex = ebpf.InterfaceUnset
-				record.Id.Direction = ebpf.DirectionUnset
```

Comment:
integration tests failed because I removed the `direction` field from this dedup method. I don't exactly understand why this field has to be unset in dedup process 🤔. Now, it is not part of the identification object `flow_id`.